net/samplicator/Makefile

```diff
@@ -1,17 +1,22 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=samplicator
-PKG_VERSION:=1.3.6
+PKG_VERSION:=2022-10-11
```

Comment:
No.

Check: https://openwrt.org/docs/guide-developer/package-policies#versioning

You are using even wrong date. In the master branch, there is latest commit from 2021.

Reply:
Typo. Thanks.